src/locales/en.json

```diff
@@ -316,41 +307,41 @@
       "instruction": "Only your face can be in the selfie"
     },
     "server_error": {
-      "message": "Connection lost",
+      "message": "$errors.server_error.message4",
       "instruction": "Please try again"
     },
     "glare_detected": {
       "message": "Glare detected",
       "instruction": "All details should be clear and readable"
     },
     "sms_failed": {
-      "message": "Something's gone wrong",
-      "instruction": "Copy the below link to your mobile instead"
+      "message": "Copy the link to your phone",
```

Comment:
All our errors follow the same structure,`message` that informs the user about what happened and `instruction` which tells the user what to do or what they did wrong. Why are we changing the format here? @raemarielee

Reply:
I switched them around on the mapping (not on the spreadsheet)